license_check.py

```diff
@@ -35,15 +36,15 @@ def check_copyright(filename, copyright_notice):
             if copyright_notice in line:
                 return True
 
-        eprint(f'No copyright notice found in {filename}')
+        eprint(f'Copyright not found in ...\t\t{filename}')
         return False
 
 
 def check_license(filename, spdx_expr):
 
     """Check if 'filename' has a SPDX identifier complying with 'spdx_exr'"""
 
-    # TODO: improve regex to support multiple licenses (AND, OR) and execeptions (WITH)
+    # TODO: improve regex to support multiple licenses (AND, OR) and exceptions (WITH)
```

Comment:
We can keep this, but note that this kind of thing should be in a separate commit.